enh(doc): prepare release notes for Centreon MBI 21.04.4 (#1679)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-commercial-extensions.md

## Centreon MBI

### 21.04.4

`July 22, 2022`

#### Bug fixes

> - Change 'switch' with 'if' to accomodate RHEL 8 deployement

Reviewer: ```suggestion
- Replaced 'switch' with 'if' to better fit RHEL 8 deployment
```

> - CBIS multiple SSH were not closed and could remain opened causing load on the server

Reviewer: ```suggestion
- Multiple SSH sessions initiated by CBIS were not closed and could remain opened causing the server overload
```
